lib/convert.ts

```diff
@@ -31,12 +30,12 @@ export default class Convert {
    *   to deal with http/https sources in the future.
    */
   public static uriToPath(uri: string): string {
-    const url = URL.parse(uri) /*eslint node/no-deprecated-api: "warn"*/ // TODO
-    if (url.protocol !== "file:" || url.path === undefined || url.path === null) {
```

Comment:
Does it become null ever?

Reply:
I don't think so but it doesn't hurt to have the check.